test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    @Test
+    public void isSimilarSubstring() throws IllegalValueException {
+        Name first = new Name("John Smith");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.